build/templates/_converters.py.mako

```diff
@@ -158,6 +158,81 @@ def convert_repeated_capabilities_without_prefix(repeated_capability):
     return ','.join(convert_repeated_capabilities(repeated_capability, ''))
 
 
+def convert_single_group_repeated_capabilities(repeated_capability):
+    '''Convert a repeated capabilities string, with at most one instrument prefix (that ends with '/') and without any comma, to a list
+
+    Examples:
+        - '0' --> ['0']
+        - '0-2' --> ['0', '1', '2']
+        - '0:2' --> ['0', '1', '2']
+        - 'Dev1/0' --> ['Dev1/0']
+        - 'Dev1/0-2' --> ['Dev1/0', 'Dev1/1', 'Dev1/2']
+        - 'Dev1/0:2' --> ['Dev1/0', 'Dev1/1', 'Dev1/2']
+
+    Args:
+        repeated_capability (str) - Supported types (with an optional instrument prefix):
+            - range (using '-' or ':')
+            - single item
+
+    Returns:
+        rep_cap (str) - comma delimited list of strings of the expanded repeated capability items
+    '''
+    if '/' in repeated_capability:
+        split_index = repeated_capability.find('/') + 1
+        instrument_prefix = repeated_capability[:split_index]
+        return convert_repeated_capabilities(repeated_capability[split_index:], instrument_prefix)
+    return convert_repeated_capabilities(repeated_capability)
+
+
+def convert_channels_repeated_capabilities(
+    channels_repeated_capability,
+    first_channel_name
+):
+    '''Convert a channels repeated capabilities string, possibly with no or multiple prefixes (each ends with '/'), to a list
+
+    Examples:
+        - convert_channels_repeated_capabilities('1', '0') --> ['1']
+        - convert_channels_repeated_capabilities('0-2', '0') --> ['0', '1', '2']
+        - convert_channels_repeated_capabilities('0:2', '0') --> ['0', '1', '2']
+        - convert_channels_repeated_capabilities('0:2,4', '0') --> ['0', '1', '2', '4']
+        - convert_channels_repeated_capabilities('4,1:2', '1') --> ['4', '1', '2']
+        - convert_channels_repeated_capabilities('2:3,0', 'Dev1/0') --> ['Dev1/2', 'Dev1/3', 'Dev1/0']
+        - convert_channels_repeated_capabilities('Dev1/1', 'Dev1/0') --> ['Dev1/1']
+        - convert_channels_repeated_capabilities('Dev1/0-2', 'Dev1/0') --> ['Dev1/0', 'Dev1/1', 'Dev1/2']
+        - convert_channels_repeated_capabilities('Dev1/0:2', 'Dev1/0') --> ['Dev1/0', 'Dev1/1', 'Dev1/2']
+        - convert_channels_repeated_capabilities('Dev1/0:2,4', 'Dev1/0') --> ['Dev1/0', 'Dev1/1', 'Dev1/2', 'Dev1/4']
```

Comment:
Is this valid syntax for the driver?

Reply:
Yes, 'Dev1/0:2,4' is a valid syntax for the driver:
```
>>> import nidcpower
>>> with nidcpower.Session(resource_name="Dev1", channels="", options="Simulate=1, DriverSetup=Model:4162; BoardType:PXIe") as smu:
...     with smu.initiate():
...         print(",".join(measurement.channel for measurement in smu.channels["Dev1/0:2,4"].measure_multiple()))
...
Dev1/0,Dev1/1,Dev1/2,Dev1/4
```

Reply:
I'm surprised that doesn't error! And I say this as the tech lead for the NI-DCPower Independent Channels tech lead.
![Screen Shot 2022-06-30 at 2 09 51 PM](https://user-images.githubusercontent.com/8085768/176758681-b5d46766-5b4d-4f9b-b1bd-745be307b6d0.png)